moon/controller.py

```diff
@@ -589,8 +704,12 @@ def drive_around_rock(self, how_far, view_direction=None, timeout=None):
             self.publish("desired_movement", [GO_STRAIGHT, 7500, -math.copysign(self.default_effort_level, how_far)]) # 1000m radius is almost straight
             self.wait(timedelta(milliseconds=100))
             if timeout is not None and self.sim_time - start_time > timeout:
-                print("go_around_a_rock - timeout at %.1fm" % distance(start_pose, self.xyz))
+                print(self.sim_time, self.robot_name, "go_around_a_rock - timeout at %.1fm" % distance(start_pose, self.xyz))
```

Comment:
maybe we should have our `self.print` function adding these two extra params ... in SubT I was using `self.stdout()` which was also publishing the info, i.e. visible in the final log